apps/extension/src/utils/background-script.utils.ts

```diff
@@ -0,0 +1,12 @@
+import { storage } from 'webextension-polyfill';
+
+export const convertLockTime = (value: number) => value * 60 * 1000;
+
+export async function fetchFromStorage<T = any>(key: string): Promise<T | null> {
```

Comment:
you are using 2 different approaches:  
1. you are calling `set` method directly where you need that
2. you did create util for `get method`

follow single approach to work with `storage`